### ISSUE TITLE
The new version v0.20.3 does not play all scheduled playlists.

### ISSUE THREAD

**Issue by Sdren** [AzuraCast/AzuraCast#7569]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

v0.20.3

### What happened?

Con la nueva actualización la mayoría de las listas de reproducción no funcionan, la nueva opción Playlist Priority no tiene sentido. Tenga mucho cuidado.

**Comment by Sdren** [comment]
he configurado varias veces y he reiniciado el sistema, pero aún no he podido volver a reproducir todas las listas

**Comment by Sdren** [comment]
I have updated doker and doker compose, I have installed interactive updates,
I have configured several times and restarted the system, but I still have not been able to play all the lists again

**Comment by BusterNeece** [comment]
@BornKillaz while I appreciate that you updated your message on here, you should know that the original contents of the message are what get e-mailed to me. I receive every comment on every issue/discussion as an e-mail in my inbox and flag them daily for follow-up if necessary.

These last few weeks have been harder than usual on me, and I've not been doing well mentally. To wake up and see your message describing "the mess that AzuraCast has become" and suggesting that we aren't working on bugfixes...it was hard enough to get out of bed this morning already, and that just...I dunno. Broke me.

I have nothing else to add. I'm going back to sleep. I may not continue working on this project.

**Comment by BornKillaz** [comment]
Hello @BusterNeece, 

I apologize if my previous comment upset you in any way. It was written in the heat of the moment, and I realize now that I should have expressed myself more carefully. 

I also understand that AzuraCast is a free, and no one has the right to demand anything. If I edited my message, it was simply because I realized I had no right to express my personal opinion in such a manner.

Once again, I’m sorry for any hurt caused. I truly wish you all the best.

**Comment by Sdren** [comment]
Hello.

I think the problem arises with the latest update. I have AzuraCast installed on Ubuntu 22.

I think AzuraCast should not lose focus. All of us who use this program think it is a very good project, and we are very grateful. You need to work in an orderly manner. I would recommend that you do not add new functions until the problems are solved.

I would like to go back to the previous version, but I don't know how to do it. I think that the Playlist Priority option is not necessary since that feature is already integrated into the lists of playback options.

I have also tried to support the project on Patreon, but it has been difficult for me since my audience is insignificant. However, I hope that the economic situation will soon improve.

**Comment by BusterNeece** [comment]
@Sdren

> You need to work in an orderly manner. I would recommend that you do not add new functions until the problems are solved.

You really shouldn't have said this.

**Comment by wegGehamstert** [comment]
> I would like to go back to the previous version, but I don't know how to do it. I think that the Playlist Priority option is not necessary since that feature is already integrated into the lists of playback options.

If you really want to get those things wich are such a big problem for you fixed, then you can do two things. 
1. Support the project trough a donation
2. Support the project trough contributing and help fixing bugs

Otherwise you are absolutly not in the position to demand stuff.

**Comment by netbladenl** [comment]
Maybe it's a language barrier but telling someone, who is doing this work almost by himself, how he should work is never right. 
This is an opensource project, which means not everything will work always as desired but we can always help with giving solutions. If you want a stable experience than you should chose the "stable' release" If you're willing to take a bit more risk and also want the newest features you should take the "rolling release".

**Comment by DJMic** [comment]
@BusterNeece You are doing a great job. Don't let negative comments get you down. Even if life is not a long quiet river. You bring very nice evolutions to Azuracast. I wanted to express my gratitude to you through this message.

**Comment by BusterNeece** [comment]
@Sdren @BornKillaz etc. I'd recommend updating to the latest stable (0.20.4) or the latest Rolling Release. Both have rolled back the scheduler code, and the Rolling Release also incorporates other fixes that bring us to a more stable state. Everything appears to be working quite normally in my local installation.

**Comment by rolradio** [comment]
Hi, First of all thank you for this great project.

Maybe it can help in the search why the playlists "x per" did not play.
Not even after the update.

I noticed that in the 'special cue' settings of the mp3 values ​​with -1 were added in the tags.
These were put in the tags of the song so that after the update from 20.3 to 20.4 they could not work.
By manually emptying these 4 fields it worked again.

![custom cue](https://github.com/user-attachments/assets/328df040-66da-4dc5-9c99-b20ff31bfb1c)
